src/object.js

```diff
@@ -140,6 +140,16 @@ function flush() {
   while ((f = delayPostFlushCallbacks.shift())) { f(); }
 }
 
+var VALID_OPTIONS = ['attr', 'converter', 'name', 'get', 'set', 'default', 'on', 'cache', 'pure'];
+// Internal: Throws error if option is passed that is not used.
+function validateOptions(opts) {
+  Object.getOwnPropertyNames(opts).forEach(function(opt) {
+    if(VALID_OPTIONS.indexOf(opt) < 0) {
```

Comment:
The O(n) operation here bugs me a little bit. This function gets called a lot when the code is initially loaded, so this could potentially have a performance impact. We could instead make `VALID_OPTIONS` an object so it would just be an O(1) lookup on each option.